src/systems/ClientSystem.ts

```diff
@@ -67,8 +95,14 @@ export class ClientSystem extends SystemWithQueries<Context> {
   start(context: Context) {
     super.start(context);
     this.resources.push(
+      this.added.onAdd((entity) => {
```

Comment:
need to keep track of entities that had the changed tag since the last save, since the ActionSystem clears the tag each frame.